docs/usage/code-coverage.mdx

```diff
@@ -7,121 +7,136 @@ description: Pester can generate code coverage metrics to help you understand ho
+   # Example output...
+   # Tests completed in 8.26s
+   # Tests Passed: 8, Failed: 0, Skipped: 0 NotRun: 0
+   # Processing code coverage result.
+   # Covered 11.7% / 75%. 735 analyzed Commands in 22 Files.
+   ```
+
+The final line displays both the current code coverage and the desired target coverage. You can customize the target coverage by configuring the CodeCoverage.CoveragePercentTarget option.
+
+## Coverage Format
 
-<#
-Code coverage report:
-Covered 60.00 % of 5 analyzed commands in 1 file.
+By default, Pester generates a `coverage.xml` file in [JaCoCo](https://www.jacoco.org/) format. You have the flexibility to modify the output format, path, and encoding by using the following options:
 
-Missed commands:
+```powershell
+$config = New-PesterConfiguration
+$config.CodeCoverage.OutputFormat = 'CoverageGutters'
+$config.CodeCoverage.OutputPath = 'cov.xml'
+$config.CodeCoverage.OutputEncoding = 'UTF8'
 
-File             Function    Line Command
-----             --------    ---- -------
-CoverageTest.ps1 FunctionOne    5 return 'SwitchParam was set'
-CoverageTest.ps1 FunctionTwo   16 return 'I do not'
-#>
+$config.CodeCoverage.Enabled = $true
 
-Invoke-Pester .\CoverageTest.Tests.ps1 -CodeCoverage @{Path = '.\CoverageTest.ps1'; Function = 'FunctionOne' }
+Invoke-Pester -Configuration $config
+```
 
-<#
-Code coverage report:
-Covered 66.67 % of 3 analyzed commands in 1 file.
+For additional `CodeCoverage` configuration options, please refer to the [New-PesterConfiguration](https://pester.dev/docs/commands/New-PesterConfiguration) documentation.
 
-Missed commands:
+> **Note:** Pester does not traverse directories. 
+If you encounter an **empty `coverage.xml` file**, it's likely because you're running the tests from a directory that doesn't
+include the relevant code to be covered. 
+To resolve this, you can either follow the recommended [Test file structure](https://pester.dev/docs/usage/test-file-structure),
+which involves placing the tests alongside the code you want to cover,
+or execute Pester from the root directory that contains both the tests and the code to be covered.
 
-File             Function    Line Command
-----             --------    ---- -------
-CoverageTest.ps1 FunctionOne    5 return 'SwitchParam was set'
-#>
+## Detailed output 
 
-Invoke-Pester .\CoverageTest.Tests.ps1 -CodeCoverage @{Path = '.\CoverageTest.ps1'; StartLine = 7; EndLine = 14 }
+When Pester is executed with detailed verbosity enabled, it provides a comprehensive output that includes detailed information about the covered lines. For example:
 
-<#
-Code coverage report:
-Covered 100.00 % of 1 analyzed command in 1 file.
-#>
+```powershell
+$config = New-PesterConfiguration
+$config.CodeCoverage.Enabled = $true
+$config.Output.Verbosity = 'Detailed'
+
+Invoke-Pester -Configuration $config
+
+# ...
+# deploy-dev-smple.ps1                                                                                            18 Write-Ou...
```

Comment:
Please format this or generate new output using a smaller console window. Too much whitespace between filename and function.